packages/x-codemod/README.md

```diff
@@ -235,4 +236,29 @@ npx @mui/x-codemod v6.0.0/data-grid/column-menu-components-rename <path>
 
 If you are using `GridRowGroupingColumnMenuItems` and `GridRowGroupableColumnMenuItems` for grouping, consider fixing them manually as these imports are replaced by `GridColumnMenuGroupingItem` and may require some extra work to port.
 
-You can find more details about Data Grid breaking change in [the migration guide](https://next.mui.com/x/migration/migration-data-grid-v5/).
+#### `row-selection-renamed-props`
+
+Data grid props that have been renamed.
+
+```diff
+  <DataGrid
+-  selectionModel={model}
++  rowSelectionModel={model}
+-  onSelectionModelChange={handler}
++  onRowSelectionModelChange={handler}
+-  disableSelectionOnClick
++  disableRowSelectionOnClick
+-  disableMultipleSelection
++  disableMultipleRowSelection
+-  showCellRightBorder
++  showCellVerticalBorder
+-  showColumnRightBorder
++  showColumnVerticalBorder
+  />
```

Comment:
Nitpick: Based on other `diff` instances it seems that using a single space should be enough and possibly more correct. Or am I missing something?
```suggestion
 <DataGrid
-  selectionModel={model}
+  rowSelectionModel={model}
-  onSelectionModelChange={handler}
+  onRowSelectionModelChange={handler}
-  disableSelectionOnClick
+  disableRowSelectionOnClick
-  disableMultipleSelection
+  disableMultipleRowSelection
-  showCellRightBorder
+  showCellVerticalBorder
-  showColumnRightBorder
+  showColumnVerticalBorder
 />
```